Codegen/test/resources/output/TypeCobol/GenContinueInsideTypdef.rdz.tcbl

```diff
@@ -22,25 +22,26 @@
       *             'zfin'.
       * 01 var1 TYPE MyType.
         01 var1.
-           02 Tit PIC X(60) value 'blbablsdfklsdfksj dfjklsz ezfrzerzer'
+           02 Tit PIC X(60) value 'blbablsdfklsdfksj dfjklsz e         z
+      -    'frzerzer'
                                .
-           02 Titr PIC X(60) value 'blbablsdfklsdfksj dfjklsz ezfrzerzer
-      -    ''
+           02 Titr PIC X(60) value 'blbablsdfklsdfksj dfjklsz e        z
+      -    'frzerzer'
                                .
-           02 Titq PIC X(60) value "blbablsdfklsdfksj dfjklsz ezfrzerzer
-      -    ""
+           02 Titq PIC X(60) value "blbablsdfklsdfksj dfjklsz e        z
+      -    "frzerzer"
                                .
-           02 Titre PIC X(60) value 'blbablsdfklsdfksj dfjklsz ezfrzerze
-      -    'r'
+           02 Titre PIC X(60) value 'blbablsdfklsdfksj dfjklsz e       z
+      -    'frzerzer'
                                .
-           02 Titre2 PIC X(60) value "blbablsdfklsdfksj dfjklsz ezfrzerz
-      -    "er"
+           02 Titre2 PIC X(60) value "blbablsdfklsdfksj dfjklsz e      z
+      -    "frzerzer"
                                .
-           02 Titre3 PIC X(60) value "blbablsdfklsdfksj dfjklsz ezfrzerz
-      -    "erzfin"
+           02 Titre3 PIC X(60) value "blbablsdfklsdfksj dfjklsz e      z
+      -    "frzerzer                                          zfin"
                           .
-           02 Titre4 PIC X(60) value 'blbablsdfklsdfksj dfjklsz ezfrzerz
-      -    'erzfin'
+           02 Titre4 PIC X(60) value 'blbablsdfklsdfksj dfjklsz e      z
+      -    'frzerzer                                          zfin'
                           .
```

Comment:
We could reduce test to 
```cob
        01 MyGroup.
           02 Titre1 PIC X(60)  value 'blbablsdfklsdfksj dfjklsz e
      -             'zfrzerzer'.
           02 Titre2 PIC X(60)  value "blbablsdfklsdfksj dfjklsz e
      -             "zfrzerzer".
           02 Titre3 PIC X(60)  value "blbablsdfklsdfksj dfjklsz e
      -             "zfrzerzer
      -             "zfin".
           02 Titre4 PIC X(60)  value 'blbablsdfklsdfksj dfjklsz e
      -             'zfrzerzer
      -             'zfin'.
```

Reply:
I don't see the redundancy. Each line is different in the input source code.

Reply:
But unless I'm missing something, in the original test the two first variables are the same except for their names. Am I missing a tab or another special character by copy/pasting directly from Git to my IDE?

Reply:
The two lines are different, they have different number of spaces included in the value.